Assignment/solution_student/style.css

```diff
@@ -0,0 +1,37 @@
+header {
+	text-align: center;
+}
+
+h1 {
+	font-size: 40px; 
+	text-transform: uppercase; 
+}
+
+h3 {
+	text-transform: uppercase;
+}
+
+body {
+	color: black;
+	font-family: sans-serif;
+    padding: 20px 0;
+    font-family: sans-serif;
```

Comment:
Oops! Looks like a duplicate line here! Just make sure to delete those when you find them :)